pkg/features/kube_features.go

```diff
@@ -1057,7 +1056,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	DisableKubeletCloudCredentialProviders: {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
 
-	DisableNodeKubeProxyVersion: {Default: true, PreRelease: featuregate.Beta},
+	DisableNodeKubeProxyVersion: {Default: false, PreRelease: featuregate.Deprecated}, // default on in 1.33
```

Comment:
Just wondering, why can't we follow the existing pattern in the KEP and return it to the alpha stage, the current approach will cause unit tests to fail and we need to modify the KEP according to it.

Reply:
alpha is not the thing failing the unit tests, defaulting back to false is what's failing, and means we have to revert the unit test changes as well

pushed an update reverting the node unit test change

Reply:
> alpha is not the thing failing the unit tests, defaulting back to false is what's failing, and means we have to revert the unit test changes as well

Sorry for my mistake, I was still working just now and only briefly looked at it (although it's already 2 AM😩). 

If everyone thinks using the deprecated gate is better, I will agree and will update the KEP with it later.

Reply:
heh, no worries, for gates which are not a new feature going through alpha/beta, we use deprecated

Reply:
> for gates which are not a new feature going through alpha/beta, we use deprecated

I 100% agree with this approach. When this KEP was implemented, we hadn't clearly defined what a deprecated gate was (at least not as clearly as we do now) :-)